File: SampleProjects/codepilot/src/Class/Food.ts
export class Food{
    public foodId: Number;
    public foodName: String;
    public amount_left: Number
    public price: Number;
    public foodTypeStatus: String;
    public imageURL: String;

    constructor(foodId: Number, foodName: string, amount_left: number, price: number, foodTypeStatus: string, imageURL: String) {
        this.foodId = foodId;
        this.foodName = foodName;
        this.amount_left = amount_left;
        this.price = price;
        this.foodTypeStatus = foodTypeStatus;
        this.imageURL = imageURL;
    }
}

File: SampleProjects/BlazorFE/BlazorApp1/Class/Food.cs
﻿namespace BlazorApp1.Class
{
    public class Food
    {
        public int FoodId { get; set; }
        public string FoodName { get; set; }
        public int AmountLeft { get; set; }
        public decimal Price { get; set; }
        public string FoodTypeStatus { get; set; }
        public string ImageUrl { get; set; }

        // Constructor
        public Food(int foodId, string foodName, int amountLeft, decimal price, string foodTypeStatus, string imageUrl)
        {
            FoodId = foodId;
            FoodName = foodName;
            AmountLeft = amountLeft;
            Price = price;
            FoodTypeStatus = foodTypeStatus;
            ImageUrl = imageUrl;
        }
    }

}